### DISCUSSION TITLE
How to set the dynamic stiffness of the mooring lines in OPENFAST？

### DISCUSSION THREAD

**Discussion by haoshuning** [OpenFAST/openfast#2407] [Q&A]
Dear Dr Jonkman,
    I am using OPENFAST to calculate the time domain response of the mooring lines. Because I am using a synthetic fiber bollard, in order to improve the accuracy of the calculation, I need to take into account its dynamic stiffness, and I would like to ask how to set this in the moordyn of OPENFAST.
   Thank you for your reply!!!!!!!!!!!

**Comment by jjonkman** [comment]
Dear @haoshuning,

@RyanDavies19 recently answered a similar question on our forum: https://forums.nrel.gov/t/some-issues-with-the-application-of-synthetic-fibers-in-floating-wind-turbines/7483.

Best regards,
